src/PatrolFilter/index.js

```diff
@@ -48,13 +51,38 @@ const PatrolFilter = ({ className }) => {
     patrolFilterTracker.track('Change Search Text Filter');
   }, []);
 
-  const resetSearch = useCallback((e) => {
-    e.stopPropagation();
+  const resetSearch = useCallback((event = null) => {
```

Comment:
nit: Seems like the `= null` is unnecessary. Adding the `?.` protection is enough.